### ISSUE TITLE
In search doesn't show original hashtag we looking for.

### ISSUE THREAD

**Issue by elaur** [mastodon/mastodon-android#356]
Hi,
I'm trying in search for specific hashtag but it is not showing. Search shows suffixes of the word for example #mastodon but not the original one. See video https://a.cl.ly/E0uZDmPo .

As can be seen in video there is not listed the main hashtag I'm looking for. The only solution to access the hashtag is to tap on some other similar hashtags and maybe I will find the hashtag I'm looking for. This is extending my time to reach what I want. I believe this is a very important UX improvement.

**Comment by realpixelcode** [comment]
That's most likely due to https://github.com/mastodon/mastodon/issues/17494

**Comment by elaur** [comment]
> That's most likely due to [mastodon/mastodon#17494](https://github.com/mastodon/mastodon/issues/17494)

Yes. Main searched hashtag should be prioritized, to be first on the list.
